### ISSUE TITLE
worrying privacy policy terms

### ISSUE THREAD

**Issue by fauno** [vector-im/riot-android#1185]
Hi, I've been looking at matrix with interest (a decentralized and encrypted messaging platform with both free software client AND server :O), but then I read [riot's privacy policy](https://riot.im/privacy) and I get worried by things like these (emphasis mine):

> When might we **disclose your information**?
> 
>   To comply with our legal obligations or to protect the interests of our users.
> 
> If we are or believe that we are under a duty to disclose or share your personal data, we will do so in order to comply with any legal obligation, **the instructions or requests of a governmental authority or regulator**, including those **outside of the UK**.
> 
> We may also choose to share information when we believe it is **necessary to detect, prevent and address illegal activity** or to otherwise protect the safety of our users.
> 
> We will share information in order to enforce or apply our Terms of Use and any other relevant agreements; or to protect our rights or property. 
> 
> [...]
>  If VECTOR CREATIONS LIMITED is sold.
> 
> In the event that we sell or buy any business or assets, **we may disclose your personal data** to the prospective seller or buyer of such business or assets.
> 
> If we or substantially **all of our assets are acquired by a third party**, personal data held by us about **our users will be one of the transferred assets**.

So a few questions:

* How does this awful privacy policy protect users' privacy?  I see it's not part of your marketing so I could be confused about this :)

* How does this privacy policy affect the matrix network?

* If I setup a matrix node, in which cases will it share (meta)data with nodes under this privacy policy?  When I talk with people with riot.im/matrix.org accounts only?  Or it will always share it? (I don't know the internal of the protocol, does it gossip data or the federation is on demand like email or jabber?)

* What about bridges to other networks such as IRC?

Thanks.

**Comment by ara4n** [comment]
@fauno the privacy policy there is just for the matrix.org (or hypothetical riot.im) homeserver - we need to make that clearer; obviously you can use the client against whatever server you like in the wider network which will have its own policy (if any).  The intention of the policy is just to comply with UK law, which is where we operate, and was written to be predictably conservative by a bunch of lawyers.  It is of course not remotely intended to undermine users' privacy, and I think that 'awful' is overstating it - the intention is that we just track the data required to run the service (i.e. conversation history; who's talking to who; etc).  The policy also reserves the right for us to track usage patterns in the app to try to determine how to improve it, although this strictly opt-in (if at all: https://github.com/vector-im/riot-web/issues/2075).

The clauses about "we may disclose personal data to an acquirer in the event of an acquisition" is just covering our ass such that if ownership of Vector/Riot ever changed, we'd be able to keep the service running rather than have to go and delete everyone's personal data (i.e. their accounts & conversation history); not because we have some evil agenda to sell out everyone's metadata or chat history to the highest bidder...

If you run your own node, then metadata will get shared with users on other servers for the rooms you share with them.  So if `@alice:alice.com` talks to `@bob:bob.com` in a 1:1 room, then the data is subject to the privacy policy of both servers.  Just like email.  There is no gossiping.  As a result, metadata is exposed on the participating homeservers (see http://matrix.org/~matthew/2016-12-22%20Matrix%20Balancing%20Interop%20and%20Privacy.pdf for the details) so if you want metadata protection, then please use Ricochet or Vuvezela/Alpenhorn or Pond or similar.

For bridging: if the bridge runs on the matrix.org HS then the data will be covered under that HS's privacy policy.  This is no different to (say) subscribing to a mailing list from your email provider and having the resulting emails stored under the privacy policy of that email provider.

So, TL;DR: if you don't like the privacy policy, please run your own server (bearing in mind that conversations shared with people on other servers will be subject to their own privacy policies).  And always e2e encrypt when you actually want privacy. 

Meanwhile, if anyone wants to pay for a bunch of lawyers to reword the policy to make it look less scary, we'd be very happy to do so - otherwise it'll probably get reworded once the matrix.org server is finally separated into an independent foundation and Riot gets its own homeserver.

**Comment by akontsevich** [comment]
I see all the communication, chat history is stored on the server (I logged in from new device while all others were offline and saw all my chats and messages). So where is the privacy then?! How it is differ from Skype which stores all the history on their servers as well since MS bought it?! Or only need to start up own server for privacy?!

**Comment by HacKanCuBa** [comment]
@akontsevich yes, that's for convenience. They rely on encryption, and pray to the encryption god, for it to provide a forever secure solution. Otherwise, if something fails, you get pwned. This isn't either good or bad, it's what it is.
Always remember: convenience = 1/security

**Comment by ara4n** [comment]
The privacy policy has been fixed as per https://github.com/vector-im/riot-meta/issues/132#issuecomment-392186296
